pkg/features/kube_features.go

```diff
@@ -107,6 +113,7 @@ var defaultKubernetesFeatureGates = map[utilfeature.Feature]utilfeature.FeatureS
 	AffinityInAnnotations:                       {Default: false, PreRelease: utilfeature.Alpha},
 	Accelerators:                                {Default: false, PreRelease: utilfeature.Alpha},
 	TaintBasedEvictions:                         {Default: false, PreRelease: utilfeature.Alpha},
+	EnableControllerTaint:                       {Default: true, PreRelease: utilfeature.Alpha},
```

Comment:
I found that most  alpha features are disabled by default, why enable this by default?